packages/backend-common/src/reading/AwsS3UrlReader.ts

```diff
@@ -70,9 +101,15 @@ export class AwsS3UrlReader implements UrlReader {
 
     return integrations.awsS3.list().map(integration => {
       const creds = AwsS3UrlReader.buildCredentials(integration);
+
       const s3 = new S3({
         apiVersion: '2006-03-01',
         credentials: creds,
+        endpoint:
+          integration.config.host === AMAZON_AWS_HOST
+            ? undefined
+            : integration.config.endpoint,
```

Comment:
Should be possible to use `integration.config.endpoint` straight away here right? since it would be undefined

Reply:
good catch, fixed